storage/ndb/src/common/portlib/NdbHW.cpp

```diff
@@ -2719,6 +2735,7 @@ static int Ndb_ReloadHWInfo(struct ndb_hwinfo * hwinfo)
   hwinfo->cpu_cnt_max = ncpu;
   hwinfo->cpu_cnt = ncpu;
   check_cpu_online(hwinfo);
+  hwinfo->is_running_in_container = 0;
```

Comment:
Is `Ndb_ReloadHWInfo` supposed to reset but not "get" the data? If so, then why do we call `check_cpu_online` that alters cpu info? If not, then we don't we call `get_meminfo`?

Also, why is this added to only one of the 6 implementations of `Ndb_ReloadHWInfo`?